fbs/gen.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+flatc -c --gen-mutable --scoped-enums --gen-object-api --no-includes torchcraft.fbs
```

Comment:
Removed the namespace post-processing seen here: https://github.com/TorchCraft/TorchCraft/blob/master/BWEnv/fbs/gen.sh#L3